src/mpid/ch4/shm/posix/posix_coll_release_gather.h

```diff
@@ -85,19 +86,25 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_POSIX_mpi_bcast_release_gather(void *buffer,
             }
         }
     }
+#ifdef HAVE_ERROR_CHECKING
+    /* When error checking is enabled, only (cellsize-cacheline_size) bytes are reserved for data.
+     * Initial cacheline_size bytes are reserved to put the amount of data being placed */
+    cellsize = MPIDI_POSIX_RELEASE_GATHER_BCAST_CELLSIZE - (2 * MPIDU_SHM_CACHE_LINE_LEN);
```

Comment:
Comment says a single cacheline_size, the code says "2", is that is a mistake?

Reply:
I fixed the comment.